backend/app/main.py

```diff
@@ -643,6 +648,106 @@ async def get_user(userid: str, current_user: dict = Depends(get_current_user)):
         raise HTTPException(
             status_code=500, detail=f"Internal Server Error. Please try again later. Error: {str(e)}")
 
+@app.get(
+    '/reviews',
+    response_model=ReviewGetResponse,
+    responses={
+        '400': {'model': ErrorResponse},
+        '404': {'model': ErrorResponse},
+        '500': {'model': ErrorResponse},
+    },
+    tags=['reviews'],
+)
+
+async def get_reviews(seller_id: str) -> Union[ReviewGetResponse, ErrorResponse]:
+    """
+    Get reviews for a seller
+    """
+    try:
+        seller = await users_collection.find_one({"_id": ObjectId(seller_id)})
+        if not seller:
+            raise HTTPException(status_code=404, detail="Seller not found")
+    
+        reviews = seller.get("reviews", [])
+        total_reviews = len(reviews)
+        average_rating = round(sum(r['rating'] for r in reviews)/total_reviews, 2) if total_reviews > 0 else 0.0
```

Comment:
Reviews should be stored in a separate collection per our DB design -- this is to ensure that we don't exceed the limit for the Mongo DB document size.